src/components/project/project.rules.ts

```diff
@@ -57,6 +58,7 @@ export interface EmailNotification {
   changedBy: Pick<User, 'id' | 'displayFirstName' | 'displayLastName'>;
   project: Pick<Project, 'id' | 'modifiedAt' | 'name' | 'step' | 'type'>;
   previousStep?: ProjectStep;
+  partnerName?: Pick<Organization, 'name'>;
```

Comment:
Since we're only using one field from here, perhaps this type should just be a `string`?  I see why you did what you did though.  I'll let @CarsonF weigh in on that.

Reply:
I agree there's a mismatch between the property type & name. I would go one way or the other to make them match.

Reply:
Technically, the property needed is the organization name, so I can just change the property name to match the actual type/property that is being retrieved.  I think I named it "partner" because the UI and requirements view it as a "partner" name.

Reply:
It's more of the fact that it's called "name" but it's an _object_ with a name.

Reply:
ah, gotcha, so just leave pass and receive an "organization" similar to how we pass a project and then pull the props out when it arrives at the template?

Reply:
I'm not sure how to change the cypher query to return the object vs the specific property in order to send the object.

Reply:
If not that way, then I could change the type from the object to a string `organizationName: string` then I don't have to mess with the cypher query. let me know which you prefer